File: codewars/count_by_x.js
// https://www.codewars.com/kata/5513795bd3fafb56c200049e

function countBy(x, n) {
  let z = [];
  z.push(x);
  for (let i = 0; i < n - 1; i++) {
    z.push(z[z.length - 1] + x);
  }
  return z;
}

File: countbyX.cpp
#include <vector>
std::vector<int> countBy(int x,int n)
{
  std::vector<int> sex(n);
  sex[0] = x;
  for(int i = 0,j = x ; i < sex.size(); ++i, j+=x)
  {
    sex[i]=j;
  }
  return sex;
}
